### ISSUE TITLE
Bug: Helm chart does not reference correct webhook version

### ISSUE THREAD

**Issue by maxgio92** [angelnu/helm-charts#54]
# Details

The suppport added recently in #49 actually does not reference the latest - to-be released, see [gateway-admission-controller#128](https://github.com/angelnu/gateway-admision-controller/issues/128) - image.

**Anything else you would like to add:**

Also, the latest Helm chart pod-gateway Helm chart's version still reference the old image (from k8s-at-home org). 

**Additional Information:**

TL;DR

- [x] release a new gateway-admission-controller version
- [ ] reference the container image tag in the pod-gateway Helm chart's `webhook.image` default values.

**Comment by maxgio92** [comment]
Hi @angelnu, I just saw it's been released a 3.8.0 version of the admission controller, thank you.
Maybe we can set this version as the default, instead of the current 3.7.0. what do you think?

**Comment by angelnu** [comment]
done @maxgio92 - https://github.com/angelnu/helm-charts/pull/59

**Comment by maxgio92** [comment]
Thank you @angelnu!
